src/App.js

```diff
@@ -0,0 +1,26 @@
+import './styles/App.css';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import Router from './routes/routes';
+import { fetchCarReservations, fetchCars } from './redux/thunk';
+
+function App() {
```

Comment:
- [ ] The images of the cars come up as null:

![image](https://github.com/tan12082001/Vehicle-Booking-App-Frontend/assets/57421823/b769f3a4-d084-4ba5-8c9f-58a36474f381)

![image](https://github.com/tan12082001/Vehicle-Booking-App-Frontend/assets/57421823/45344efb-e9b6-4943-8080-86acef81e4e2)

Kindly, make it so the images of the cars show up. 👍